.template/versions.tf

```diff
@@ -0,0 +1,13 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">=4.0.0"
+    }
+    kubernetes = {
+      source  = "hashicorp/kubernetes"
+      version = ">=2.0.0"
+    }
+  }
+  required_version = ">= 0.14"
```

Comment:
Should use a more recent version here

Reply:
have updated in other branch which will be merged before this